mozilla/blurts-server
# Don't send new users the monthly activity email

# References: 
Jira: MNTOR-3186
Figma: N/A




# Description

The first email should only get sent after the user actually has activity, i.e. after the first month.

# How to test

Create a new account, subscribe them to Plus (ensure they have the susbcription in the DB via `/admin/dev`). Make sure the `MonthlyActivityEmail` feature flag is enabled for them. Run the `npm run dev:cronjobs` script. It should not send the email to them - but when you switch to `main`, it does.

# Checklist (Definition of Done)
- [ ] I've added a unit test to test for potential regressions of this bug. - N/A, DB query
